usecases/scheduler/src/lambda/ssv2_configUpdater/configUpdater.py

```diff
@@ -0,0 +1,274 @@
+    if event['Records'][0]['eventName'] in ('INSERT', 'MODIFY'):
+        items = sanitizeEvent(event, 'INSERT')
+        for item in items:
+            configId = item['configId']
+            emails = item['emails']
+            ssparams = item['ssparams']
+            cronPattern = item['frequency']
+            crossAccounts = item['crossAccounts']
+
+            print('Patching the following Config: {}'.format(configId))
+            result = updateSnsRecipient(configId, emails)
+            if result == False:
+                msg = 'Fail to update SNS - email recipients'
+                resp = {'statusCode': 500, 'body': msg}
+                return resp
+
+            result = updateEventBridge(configId, ssparams, cronPattern, crossAccounts)
+            if result == False:
+                msg = 'Fail to update eventBridge Configuration for: {}'.format(configId)
+                resp = {'statusCode': 500, 'body': msg}
+                return resp
+
+    # If event is Delete item from DDB
+    else:
+        items = sanitizeEvent(event, 'REMOVE')
+        for item in items:
+            configId = item['configId']
+            print('Deleting the following Config: {}'.format(configId))
+
+            result = deleteEventBridge(configId)
+            if result == False:
+                msg = 'Failed to delete eventBridge Configuration for: {}'.format(configId)
+                resp = {'statusCode': 500, 'body': msg}
+                return resp
+
+            result = deleteSnsRecipient(configId)
+            if result == False:
+                msg = 'Failed to delete SNS topic for: {}'.format(configId)
+                resp = {'statusCode': 500, 'body': msg}
+                return resp
+    
+    return successResp
+    
+
+## update EventBridge
+def updateEventBridge(ssv2configId, ssparams, cronPattern, crossAccounts):
+    ## check if schedule exists
+    inputJson = { 
+        "JobDefinition": jobDef, 
+        "JobName": "ssv2-" + ssv2configId, 
+        "JobQueue": jobQueue, 
+        "ContainerOverrides": 
+            { "Environment": 
+                [ 
+                    { "Name": "PARAMS", "Value": ssparams }, 
+                    { "Name": "S3_OUTPUT_BUCKET", "Value": s3Bucket },
+                    { "Name": "CONFIG_ID", "Value": ssv2configId },
+                    { "Name": "CROSSACCOUNTS", "Value": crossAccounts}
+                ] 
+            } 
+    }
+
+    target = {
+        'Arn': 'arn:aws:scheduler:::aws-sdk:batch:submitJob',
+        'RoleArn': ebrolesARN,
+        'Input': json.dumps(inputJson)
+    }
+
+    flexibleTimeWindow = {
+        "Mode": 'OFF'
+    }
+
+    args = {
+        "Description": '[Created by SSv2] Screener Scheduler:' + ssv2configId,
+        "FlexibleTimeWindow": flexibleTimeWindow,
+        "Name": 'ScreenerScheduler-' + ssv2configId,
+        "ScheduleExpression": cronPattern,
+        "ScheduleExpressionTimezone": 'UTC',
+        "StartDate": datetime.now(),
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The naive datetime objects are treated by many datetime methods as local times, it is preferred to use aware datetimes to represent times in UTC. The recommended way to create an aware datetime object representing a specific timestamp in UTC is by passing `tzinfo` as an argument to the method.

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)